### DISCUSSION TITLE
Mesh Vertices 프로퍼티 참조

### DISCUSSION THREAD

**Discussion by xowjd913** [naverz/zepeto-studio-kor#125] [5. Q&A]
안녕하세요.

sharedMesh.vertices 로 버텍스 어레이를 가져오면 undefined 로 값을 가져와서 사용이 불가능 합니다.
혹시 다른방법으로 버텍스 어레이를 가져올수 있는 방법이 있을까요 ?

![image](https://user-images.githubusercontent.com/25768804/147182548-a32c2e21-e700-4890-8b4f-0dffae36b647.png)

현재 코드입니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 이슈에 대해 도움을 드리고 싶은데
전체 코드 및 스크립트가 들어있는 오브젝트 구조 스크린샷을 올려주시면 감사하겠습니다.
